x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -250,6 +250,17 @@ export function App({
         ? i18n.translate('xpack.lens.breadcrumbsByValue', { defaultMessage: 'Edit visualization' })
         : persistedDoc.title;
     }
+    if (
+      !persistedDoc?.title &&
+      initialContext &&
+      initialContext.originatingApp === 'dashboards' &&
```

Comment:
I was hoping that we could omit using "dashboards" as check here. This is also visible in canvas. We need another approach here and I know that @ThomThomson is against it.

Reply:
Removed it